### PR TITLE
TINY-10885: Keep focus on Image Dialog after image upload.

### PR DESCRIPTION
**Pull request by vpyshnenko**
Related Ticket: TINY-10885

Description of Changes:
* Set focus on the image Source input field after image upload

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Edit image modal did lose focus after image upload.
```
Comment (TheSpyder) [inline .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml:3]: "edit image" is a feature now, so we probably need to be more specific
```suggestion
body: Insert/Edit image dialog lost focus after the image upload completed.
```

**Review by TheSpyder — APPROVED** [review]

**Review by spocke — APPROVED** [review]

**Review by hamza0867 — APPROVED** [review]

File: .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Insert/Edit image dialog lost focus after the image upload completed.
```
Comment (hamza0867) [inline .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml:3]: I think we would usually use the `would` word in here so something like:
```suggestion
body: Insert/Edit image dialog would lose focus after the image upload completes.
```

Comment (TheSpyder) [reply .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml:3]: The only real requirement is that it's past tense - so what I wrote and your suggestion are both valid. It comes down to personal choice.

Comment (vpyshnenko) [reply .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml:3]: @TheSpyder, could you please revise this improvement as a native speaker?

Comment (TheSpyder) [reply .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml:3]: @vpyshnenko I prefer my suggestion, that's why I wrote it :)

If we think it needs rewording, how about
```suggestion
body: The Insert/Edit image dialog no longer loses focus after image uploads complete.
```

Comment (vpyshnenko) [reply .changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml:3]: I'll stick your original one since it is in the past tense

**Pull request merged**